spec/controllers/tags_controller_spec.rb

```diff
@@ -173,6 +173,66 @@
     end
   end
 
+  describe "show_hidden" do
+    render_views
+
+    before do
+      @fandom1 = FactoryBot.create(:fandom, canonical: true)
```

Comment:
- You should use `let` instead of instance variables: http://www.betterspecs.org/#let.
- House style: we usually leave out `FactoryBot.` in RSpec tests.

We don't do either consistently yet, but we prefer them for new tests. You don't have to change existing ones, though.

Reply:
Thanks, will do!

Reply:
Ah crud, I missed this in the test style docs - sorry, I'm just learning Ruby so I didn't have the context for that the first time I saw it. I'll have a look through those.

Reply:
Resolved in 40e540a5ab909648b65b259ee99ec8c10de550b6